TestPlaybooks/playbook-IPInfo_PY_Test.yml

```diff
@@ -0,0 +1,194 @@
+id: IPInfoTest
+version: 5
+name: IPInfo PY
+starttaskid: "0"
+tasks:
+  "0":
+    id: "0"
+    taskid: 4a0483a3-542a-4e66-8c95-6e80f24dd619
+    type: start
+    task:
+      id: 4a0483a3-542a-4e66-8c95-6e80f24dd619
+      version: -1
+      name: ""
+      iscommand: false
+      brand: ""
+    nexttasks:
+      '#none#':
+      - "1"
+    separatecontext: false
+    view: |-
+      {
+        "position": {
+          "x": 480,
+          "y": 50
+        }
+      }
+    note: false
+  "1":
+    id: "1"
+    taskid: 98c652e0-eace-4244-8b5d-8e9224193007
+    type: regular
+    task:
+      id: 98c652e0-eace-4244-8b5d-8e9224193007
+      version: -1
+      name: DeleteContext
+      description: Delete field from context
+      scriptName: DeleteContext
+      type: regular
+      iscommand: false
+      brand: ""
+    nexttasks:
+      '#none#':
+      - "2"
+    scriptarguments:
+      all:
+        simple: "yes"
+      key: {}
+    separatecontext: false
+    view: |-
+      {
+        "position": {
+          "x": 480,
+          "y": 195
+        }
+      }
+    note: false
+  "2":
+    id: "2"
+    taskid: 592fa11b-fef4-4b83-82dc-88015c27572d
+    type: regular
+    task:
+      id: 592fa11b-fef4-4b83-82dc-88015c27572d
+      version: -1
+      name: ip
+      description: Check IP reputation (when information is available, returns a JSON
+        with details).  Uses all configured Threat Intelligence feeds
+      script: IPInfo Test|||ipinfo
+      type: regular
+      iscommand: true
+      brand: IPInfo Test
```

Comment:
brand and script names are not correct. your tests have passed since this playbook wasn't added to the test conf.
we'll go through this next week (remind me)